From 4faa5acf9ca0c777c7ae21080d4afa05030221b4 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Thu, 6 Dec 2018 16:05:24 -0800
Subject: [PATCH] MANTA-4030 many images using node 0.10 or 0.12 broken by
 transitive lru-cache@4.1.4 dep (pinning lru-cache version is no longer
 necessary after 4.1.5 release) Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
 Approved by: Josh Wilsdon <josh@wilsdon.ca>

---
 package.json | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/package.json b/package.json
index 9beab2b..98f696c 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "fwapi",
   "description": "Firewall API for Smart DataCenter",
-  "version": "1.0.6",
+  "version": "1.0.7",
   "author": "Joyent",
   "private": true,
   "devDependencies": {
@@ -12,7 +12,6 @@
     "eslint-plugin-joyent": "~1.3.0",
     "mockery": "1.4.0"
   },
-  "// lru-cache": "attempting to pin version while satisfying transitive deps, because lru-cache@4.1.4 includes yallist@3 which breaks node --abort-on-uncaught-exception with node <v4",
   "dependencies": {
     "assert-plus": "1.0.0",
     "bunyan": "1.8.10",
@@ -23,7 +22,6 @@
     "fast-messages": "1.0.1",
     "fwrule": "2.0.0",
     "jsprim": "1.3.1",
-    "lru-cache": "4.1.3",
     "moray": "2.0.1",
     "moray-filter": "1.0.0",
     "tape": "4.2.2",
-- 
2.21.0

